operator/internal/manifests/internal/config/loki-runtime-config.yaml

```diff
@@ -58,8 +58,7 @@ overrides:
     blocked_queries:
     {{- range $blockedQuery := . }}
     - {{ with $blockedQuery.Pattern -}}
-      pattern: |
-        {{ . | yamlBlock "        "  }}
+      pattern: {{ . | printf "%q" }}
```

Comment:
(can't attach a suggestion to the whole block)

Looking at this configuration block again, I noticed that this is another one of those blocks where every attributes is optional (even though at least one should be present in reality). I wonder if we should make this more robust by rendering at least one attribute in all cases, so that YAML parse errors should not occur. The only sensible choice for me seems to be the `regex` field, because it can be `false` without affecting anything.

@periklis Do you know if Loki treats blocked queries having `regex=false` differently from having `regex` unset? If they are treated the same, I would propose this improvement:

```
    - regex: {{ $blockedQuery.Regex }}
      {{ with $blockedQuery.Pattern -}}
        pattern: {{ . | printf "%q" }}
      {{ end -}}
      {{- with $blockedQuery.Types }}
      types: {{ . }}
      {{- end }}
      {{- with $blockedQuery.Hash }}
      hash: {{ . }}
      {{- end }}
```

Reply:
AFAICS there is no notion on unset regex in the Loki code base: https://github.com/grafana/loki/blob/main/pkg/logql/blocker.go#L55-L66

I agree we should always set at least a single field, but the best option is `pattern: ''` because setting `regex:false` by default will be translated as `pattern:"" regex: true` in the Loki code base. WDYT?

Reply:
During the daily we came to the conclusion, that it probably does not make a difference _technically_ if we use `regex` or `pattern`, but the empty pattern is more obvious to the casual onlooker, so we settled on using that property instead :slightly_smiling_face: